src/components/app-registration/AppManagement/AppManagement.jsx

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable react/jsx-key */
-
+import React from "react";
```

Comment:
@hubert-deriv let's create a new PR to impose single quotes for all the jsx files.

You can add 

> "singleQuote": true,
> "jsxSingleQuote": true,

to `.prettierrc` file.

```suggestion
import React from 'react';
```

Reply:
Good point, will make a new PR for that 👍